config/vanilla/config.php

```diff
@@ -121,8 +121,8 @@
 $Configuration['Plugins']['Topcoder']['SSO']['TopcoderHS256']['Secret'] =  getenv('TOPCODER_HS256_SECRET');
 $Configuration['Plugins']['Topcoder']['SSO']['TopcoderRS256']['UsernameClaim'] =  'nickname';
 $Configuration['Plugins']['Topcoder']['SSO']['TopcoderHS256']['UsernameClaim'] =  'handle';
-$Configuration['Plugins']['Topcoder']['SSO']['TopcoderHS256']['UserIDClaim'] = 'https://topcoder.com/userId';
-$Configuration['Plugins']['Topcoder']['SSO']['TopcoderRS256']['UserIDClaim'] = 'https://topcoder.com/userId';
+$Configuration['Plugins']['Topcoder']['SSO']['TopcoderHS256']['UserIDClaim'] = getenv(('TOPCODER_PLUGIN_SSO_TOPCODER_USERID_CLAIM'));
```

Comment:
is this missing `/userId` part?

Reply:
@kkartunov this is how the env var looks like:
`"TOPCODER_PLUGIN_SSO_TOPCODER_USERID_CLAIM": "https://topcoder-dev.com/userId",`